multiuser/machine-auth/che-multiuser-machine-authentication/src/main/java/org/eclipse/che/multiuser/machine/authentication/server/MachineAuthModule.java

```diff
@@ -79,17 +79,12 @@ protected void configure() {
         .toInstance(
             new MachineAuthenticatedResource(
                 "/preferences", "find", "save", "update", "removePreferences"));
-    machineAuthenticatedResources
-        .addBinding()
-        .toInstance(new MachineAuthenticatedResource("/user", "getCurrent"));
     machineAuthenticatedResources
         .addBinding()
         .toInstance(new MachineAuthenticatedResource("/activity", "active"));
     machineAuthenticatedResources
         .addBinding()
-        .toInstance(
-            new MachineAuthenticatedResource(
-                "oauth", "token", "authenticate", "callback", "getRegisteredAuthenticators"));
+        .toInstance(new MachineAuthenticatedResource("oauth", "token"));
```

Comment:
`.toInstance(new MachineAuthenticatedResource("oauth", "token"));` this line can be removed too?